### ISSUE TITLE
Receiving 404 when trying to download 3.7.5 binary

### ISSUE THREAD

**Issue by dannypaz** [CircleCI-Public/serverless-framework-orb#23]
**Orb Version**

circleci/serverless-framework@1.0.1

**Describe the bug**
build is currently failing due to 404 on version 3.7.5 of serverless

```
 Downloading binary for version 3.7.5...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   657  100   657    0     0   9385      0 --:--:-- --:--:-- --:--:--  9385
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 
Could not download binary. Is the version correct?
Exited with code exit status 22
```

**To Reproduce**

Use orb and run `- serverless/setup` in your config

```
orbs:
  serverless: circleci/serverless-framework@1.0.1

jobs:
  build:
    working_directory: ~/MyApp
    executor: serverless/default
    resource_class: large
    docker:
      - image: cimg/node:14.17.1
    steps:
      - checkout
      - serverless/setup
```
      

**Expected behavior**

No failures when trying to download latest version of serverless (3.7.5)

**Additional context**

Serverless framework just releases version 3.7.5 this morning (maybe 3 hours ago from this post, March 14) https://github.com/serverless/serverless/releases/tag/v3.7.5

**Comment by krishnagopinath** [comment]
Facing the same issue as well.

**Comment by biancabarria** [comment]
Facing the same issue as well.

**Comment by KyleTryon** [comment]
Hello,
Thank you all for reporting this issue. We have just released version 2.0.0 which should resolve this problem. Please try the latest version of the orb and let us know if you experience any issues.

https://github.com/CircleCI-Public/serverless-framework-orb/releases/tag/v2.0.0
https://circleci.com/developer/orbs/orb/circleci/serverless-framework
